openshift/auth.config.yaml

```diff
@@ -20,9 +20,9 @@ objects:
     github.client.id: Cg==
     github.client.secret: Cg==
     oso.client.apiurl: aHR0cHM6Ly9hcGkuY29uc29sZS5zdGFydGVyLXVzLWVhc3QtMi5vcGVuc2hpZnQuY29t
-    oso.client.id: Cg==
-    oso.client.secret: Cg==
     tenantsvc.url: aHR0cDovL2Y4dGVuYW50Ojgw
+    oso-clusters: >-
```

Comment:
Can you please change this to `oso.cluster` to keep the naming of other keys?

Reply:
Good point! Created a new PR for that - https://github.com/fabric8-services/fabric8-auth/pull/197